warhorn/warhorn.py

```diff
@@ -1198,12 +1181,25 @@ def assemble_warrior():
 
     check_basic_requirements(logfile, config_file_name, console_log_name,
                              print_log_name, python_executable)
-    get_dependencies(logfile, print_log_name, config_file_name)
+    node = get_node(config_file_name, 'virtualenv')
+    if node is not False and get_attribute_value(node, 'activate') == 'yes':
+        if get_attribute_value(node, 'install') == 'yes':
+            install_depen('virtualenv==15.1.0', 'virtualenv', logfile, print_log_name)
+        war_tag = get_node(config_file_name, 'warriorframework')
+        if war_tag is False:
+            print_error("warriorframework is a mandatory repo. Please add and rerun",
+                        logfile, print_log_name)
+            setDone(1)
+            getDone(logfile, print_log_name)
+        dest = get_attribute_value(war_tag, 'destination')
+        virtualenv_activated = activate_virtualenv(node, dest, logfile, print_log_name)
```

Comment:
I am looking for the something similar to
https://stackoverflow.com/questions/6943208/activate-a-virtualenv-with-a-python-script
under
"""
However if you want to activate the virtualenv under the current python interpreter...
"""